### DISCUSSION TITLE
Dead Link in Plugins List

### DISCUSSION THREAD

**Discussion by moritztim** [remarkjs/remark#1386] [General] [answered]
https://github.com/remarkjs/remark/blob/1334327f4d0289dbd0452d89060081c18f6abfab/doc/plugins.md?plain=1#L204

This repo doesn't exist, maybe it's supposed to be this? https://github.com/atinux/remark-macro

**Comment by wooorm** [comment, marked as answer]
Hi!

Sometimes people remove things from GitHub.

It looks like the `atinux` version is behind (1.0.3 instead of 1.0.7). I would guess that it was a fork which GH “disconnects” when the upstream repo was removed.

I removed it https://github.com/remarkjs/remark/commit/18ff827963f57d1becbc1e629f8885b03eeb85a3
